sass/site/primary/_posts-and-pages.scss

```diff
@@ -168,7 +168,6 @@
 				height: 100%;
 				top: 0; left: 0;
 				content: "\020";
-				display: block;
```

Comment:
Is this left over from a prior commit? Otherwise, this looks good!

Reply:
Yeah it's unrelated, I just happened to notice it and removed it. There's already a display: block in line 166.

<img width="219" alt="captura de pantalla 2018-10-29 a la s 12 54 32" src="https://user-images.githubusercontent.com/1041600/47662513-d1ab7280-db79-11e8-8962-3b62a51bcd69.png">

Reply:
Confirmed. 👍 Thanks for the fix.